Costume/100_Voltorb.yaml

```diff
@@ -0,0 +1,20 @@
+species: Voltorb
+imagename: Wooloo
+displayname: Wooloo
+setname: Costume
+tags: [costume]
+item: [Normal Gem]
+ability: [Run Away]
+moves:
+    - [Take Down]
+    - [Stomp]
+    - [Defense Curl]
+    - [Rest]
```

Comment:
recovery, def booting and not being able to hit ghosts sounds like a bad combo

Reply:
Wouldn’t Rollout work here because, y’know, Wooloo?

Reply:
Wooloo can't legally learn Rollout for whatever reason. Considering I was basing this off what moves it DOES have that means Rollout wouldn't have a chance.

Yes, it literally doesn't make sense why it wouldn't have it.

Reply:
wooloo lacks options, you have payback or.... payback.

Reply:
Yeahhhh kinda sucks :/ He gains some electric type moves thru TMs but those both come from Gen V.
I added Double Kick in place of Rest to offset one of the issues with the set tho. Might just have to deal with it for the time being until we can add those moves in.